include/sys/zfs_project.h

```diff
@@ -60,6 +60,9 @@ typedef struct zfsxattr zfsxattr_t;
 #define	ZFS_IOC_FSSETXATTR	_IOW('X', 32, zfsxattr_t)
 #endif
 
+#define	ZFS_IOC_GETDOSFLAGS	_IOR(0x83, 1, uint64_t)
+#define	ZFS_IOC_SETDOSFLAGS	_IOW(0x83, 2, uint64_t)
```

Comment:
Was `0x83` chosen specifically to not conflict, or is there some other significance here?

I'd also suggest moving these to `include/sys/fs/zfs.h` under an `#ifdef __linux__` guard so they're available to both user and kernel space.  Then you could include this header in your test programs.  Moving the `ZFS_READONLY,etc` flags too would be good if we're now exposing these bits to user space.

Reply:
`0x83` was chosen specifically to not conflict. There is no other significance.
Sure, I'll move these changes to `include/sys/fs/zfs.h`, as suggested.